xbmc/addons/kodi-dev-kit/include/kodi/gui/General.h

```diff
@@ -28,6 +27,20 @@ namespace gui
 /// to enjoy it.
 ///
 
+//==============================================================================
+/// @ingroup cpp_kodi_gui_general
+/// @brief **Adjust refresh rate enum**\n
+/// Used to get the Adjust refresh rate status info.
+///
+enum AdjustRefreshRateStatus
+{
+  ADJUST_REFRESHRATE_STATUS_OFF = 0,
+  ADJUST_REFRESHRATE_STATUS_ALWAYS,
+  ADJUST_REFRESHRATE_STATUS_ON_STARTSTOP,
+  ADJUST_REFRESHRATE_STATUS_ON_START,
```

Comment:
Looks good here, only as the base is an "C" should it better on https://github.com/xbmc/xbmc/blob/master/xbmc/addons/kodi-dev-kit/include/kodi/c-api/gui/general.h

Reply:
moved